ui-icons/README.md

```diff
@@ -10,10 +10,7 @@ Based upon https://octicons.github.com our icons set provide slick & clear picto
 ## To add a new Icon
 
 1. import svg in `icons` folder
-2. in osrd-ui/ui-icons:
-
-- launch `npm run svgo` to optimize svg size.
-- launch `npm run build-components`
```

Comment:
I think `npm run build` does this now?

Reply:
Well, it's required to bundle the new icon inside `dist/`. Ah, the next step mentions `npm run build` anyways, so we can indeed drop this bullet point safely.